README.md

```diff
@@ -1 +1,202 @@
-# Hello-world
\ No newline at end of file
+<a name="readme-top"></a>
+
+
+<!-- TABLE OF CONTENTS -->
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
```

Comment:
Well done on using the README template 📖 but make sure that the `Table of Contents` includes all of the sections of the Template and that the body of the README includes all of them as well, you're only allowed to remove the `FAQ` because it's **OPTIONAL** 🤷🏽 , the `Live Version` because it does not apply to this project, for the other sections such as `Run Tests` and `Deployment` you may also remove them or specify that they do not apply. ❌ As for the `Usage` you could specify to open the project in the browser 🌐 . Take a look at this [one](https://github.com/mohamedgamaleldin999999/The-Jimmy-Project/blob/main/README.md) for example. 💯